includes/functions.php

```diff
@@ -1180,6 +1180,10 @@ function ap_sanitize_unslash( $str, $from = false, $default_val = '' ) {
 		}
 	}
 
+	if ( '0' == $str ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
@rahularyan 
Refer to this code here: https://github.com/anspress/anspress/blob/16c59942049b49b58fec8e13bf87ec7aa1062b5c/addons/notifications/notifications.php#L235
In this code, the below function is called: https://github.com/anspress/anspress/blob/16c59942049b49b58fec8e13bf87ec7aa1062b5c/includes/functions.php#L1173

However, since the template file to display the notification passes the request value as **0** from here: https://github.com/anspress/anspress/blob/16c59942049b49b58fec8e13bf87ec7aa1062b5c/templates/addons/notification/index.php#L36
so this condition will be called since **0** is treated as empty: https://github.com/anspress/anspress/blob/16c59942049b49b58fec8e13bf87ec7aa1062b5c/includes/functions.php#L1187-L1189
which will return the default value passed, which we're not doing for this case. So, in order to meet this case I've introduced if for the required result.

Reply:
But instead of making change to that core function can you please add your adjustments directly to notifications.php?

Reply:
@rahularyan Adjusted the required changes on the `notifications.php` file.